gradle/build-logic/check-dexer/build.gradle.kts

```
@@ -0,0 +1,4 @@
+plugins {
+    base
+    id("build-logic.kotlin-dsl-gradle-plugin")
+}
```

```suggestion
}

dependencies {
    api(projects.basics) 
}
```
basics adds the Project.prefixedProject extension

---

nice, thanks!